.github/workflows/build-push.yml

```diff
@@ -61,11 +62,8 @@ jobs:
             ${{ runner.os }}-go-
       - uses: actions/setup-go@v3
       - run: make -e vendor
-      - run: |
-          echo "ref: ${{ github.ref }}"
-          echo "ref_name: ${{ github.ref_name }}"
       - uses: docker/login-action@v2
-        if: ${{ github.event_name == 'push' && github.ref_name == 'main' && matrix.builder == 'docker' }}
+        if: ${{ github.event_name == 'push' && matrix.builder == 'docker' }}
```

Comment:
@okartau is there a reason for the `event_name` check since this Action triggers on `push` only?

Reply:
indeed, in context of this PR, event_name is not needed.
Why I keep it that this push idea started as added part in CI flow, later I detached push part from there to keep the devel work isolated. If we want to embed push part into CI flow again, then we will need event name check again, so I wanted to keep it with code as reminder that we need it there again.

Reply:
I don't mind keeping it separate here as it is for now also because we don't publish/push the same set of images as in what we build/test in CI. We could drop `if:` conditionals and build with `docker` only. I'm OK keeping the "TAG value in Makefile" for now as I don't have good idea how the future releasing will look like.